torch_frame/data/multi_embedding_tensor.py

```diff
@@ -117,6 +94,64 @@ def from_tensor_list(
         offset = torch.LongTensor(offset_list)
         return cls(num_rows, num_cols, values, offset)
 
+    def __getitem__(
+        self,
+        index: Any,
+    ) -> Union["MultiEmbeddingTensor", Tensor]:
+        if isinstance(index, tuple) and len(index) == 2 and isinstance(
```

Comment:
If the tuple is `(1, '4')` then this function will raise a `NotImplementedError`, right?

Reply:
Can you change it to raise a different error if the format of the Tuple is wrong? Thanks!

Reply:
I think it is fine for now. The feature is still being developed, and when we get feature-complete, let's raise an informative error, like xxx indexing is not supported.